src/pybind/mgr/volumes/fs/operations/versions/subvolume_v2.py

```diff
@@ -337,14 +337,14 @@ def open(self, op_type):
         except cephfs.Error as e:
             raise VolumeException(-e.args[0], e.args[1])
 
-    def trash_incarnation_dir(self):
+    def trash_incarnation_dir(self, subvol_path):
         """rename subvolume (uuid component) to trash"""
         self.create_trashcan()
         try:
-            bname = os.path.basename(self.path)
+            bname = os.path.basename(subvol_path)
             tpath = os.path.join(self.trash_dir, bname)
-            log.debug("trash: {0} -> {1}".format(self.path, tpath))
-            self.fs.rename(self.path, tpath)
+            log.debug("trash: {0} -> {1}".format(subvol_path, tpath))
```

Comment:
Using f-strings instead of str.format() is always better.
```suggestion
            log.debug(f"trash: {subvol_path} -> {tpath}")
```